.gitignore

```diff
@@ -1,15 +1,19 @@
+# THERE IS ANOTHER .gitignore IN test/sql!
+
 .*.swp
 pgtap.sql
 pgtap-core.sql
 pgtap-schema.sql
 uninstall_pgtap.sql
-test/setup.sql
```

Comment:
Why is this deleted; it's  a generated file. Unless you mean to put it into `test/.gitignore`?